src/web/RestApi.h

```diff
@@ -929,6 +929,7 @@ class RestApi {
 
         void getDisplay(JsonObject obj) {
             obj[F("disp_typ")]          = (uint8_t)mConfig->plugin.display.type;
+            obj[F("disp_motion")]       = (boolean)mConfig->plugin.display.motion;
```

Comment:
Wie oben: mConfig = Config, nicht Live-Daten.
Die Motion Live-Daten könnte man meiner Meinung nach in den Generic Teil der Rest API geben,  da sind auch schon ein paar andere drin, die sonst nirgendwo recht dazu passen.